aries-site/src/pages/components/button.mdx

```diff
@@ -381,3 +379,19 @@ For button groups on screen widths less than or equal to 576px (Grommet "small")
     <ButtonBadGroupPreview />
   </Example>
 </BestPracticeGroup>
+
+## Accessibility
+### A11y guidance
+The following guidance should be considered for each instance of the component and is the responsibility of the implementer. WCAG criteria provided directly by the Grommet component are summarized under [WCAG compliance](#wcag-compliance).
+
+**Note**: WCAG rules marked as conditional indicate that the implementer must take additional actions to ensure full compliance.
```

Comment:
Would this "Note" be better executed as an inline Notification? More prominence so it doesn't get lost?

Reply:
Good idea, I changed this to be an inline notification and also moved it under he WCAG compliance section, seemed like a better fit for it to live there